### ISSUE TITLE
Tectonic with miniconda: biber and biblatex versions are incompatible

### ISSUE THREAD

**Issue by cschaf** [PHPirates/travis-ci-latex-pdf#26]
`ERROR - Error: Found biblatex control file version 3.4, expected version 3.1.
This means that your biber (2.5) and biblatex (3.11) versions are incompatible.
See compat matrix in biblatex or biber PDF documentation.
INFO - ERRORS: 1`

Is that error known? How could i solve it?

**Comment by PHPirates** [comment]
Aha, it looks like the package version that Tectonic downloads is not compatible any more with the biber version installed manually, this is because the Tectonic packages have updated but we didn't update the biber version in the config. These lines from your config are relevant:

```
      conda install tectonic==0.1.10;
    fi
  # Install biber if required for compiling the references.
  # Biber 2.5 is required for tectonic 0.1.8
  - conda install -c malramsay biber==2.5 --yes
```

The question is now what we should change the numbers into. The latest [tectonic](https://tectonic-typesetting.github.io/en-US/) is 0.1.11, and [latest biber](https://ctan.org/pkg/biber?lang=en) seems to be 2.13. The [biber docs](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=2ahUKEwib78Kw3uvkAhUEaVAKHbmpChEQFjAAegQIAhAB&url=http%3A%2F%2Fmirrors.ctan.org%2Fbiblio%2Fbiber%2Fdocumentation%2Fbiber.pdf&usg=AOvVaw38DYKJ7HPMsYodgrIYKROm) compatibility matrix shows that biblatex 3.11 is compatible with biber 2.11.


Can you try changing 2.5 to 2.11 in your config and try again?

**Comment by cschaf** [comment]
It worked! I just changed the biber==2.5 to biber==2.11 and it worked!

**Comment by PHPirates** [comment]
I see it, great! I will update the config, and will add instructions on how to find the correct magic version numbers.

**Comment by PHPirates** [comment]
Fixed in ed19d266fb6873b192f8e930aeb0adb850fd8b03, added it to the tests so I will get notified when it has to be changed again.
